roachtest: hibernate failed

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3473224&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3473224&tab=artifacts#/hibernate) on release-21.1 @ [c2cc9747352c2c1e95b4ed9f3ef3d34d2f7d1257](https://github.com/cockroachdb/cockroach/commits/c2cc9747352c2c1e95b4ed9f3ef3d34d2f7d1257):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-36-gc2cc974735
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3479941&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3479941&tab=artifacts#/hibernate) on release-21.1 @ [8a84d042ae5ba3639b6e8a82adc831539b48668c](https://github.com/cockroachdb/cockroach/commits/8a84d042ae5ba3639b6e8a82adc831539b48668c):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-42-g8a84d042ae
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3490931&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3490931&tab=artifacts#/hibernate) on release-21.1 @ [bdc1360164aa2a81aedf3b8c45c43389505eeeca](https://github.com/cockroachdb/cockroach/commits/bdc1360164aa2a81aedf3b8c45c43389505eeeca):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-46-gbdc1360164
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3507405&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3507405&tab=artifacts#/hibernate) on release-21.1 @ [27054e236beb375be3ca06b824d18b0d503f3205](https://github.com/cockroachdb/cockroach/commits/27054e236beb375be3ca06b824d18b0d503f3205):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-57-g27054e236b
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3513116&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3513116&tab=artifacts#/hibernate) on release-21.1 @ [5bd1a16edef216dd764b6ac5ff2974d894e3dbbe](https://github.com/cockroachdb/cockroach/commits/5bd1a16edef216dd764b6ac5ff2974d894e3dbbe):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-64-g5bd1a16ede
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3518596&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3518596&tab=artifacts#/hibernate) on release-21.1 @ [ac6f714445f280f7ce0f867ae54ac10ffbeaf984](https://github.com/cockroachdb/cockroach/commits/ac6f714445f280f7ce0f867ae54ac10ffbeaf984):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-66-gac6f714445
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3524681&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3524681&tab=artifacts#/hibernate) on release-21.1 @ [3a28e789f6739906f0ebe333c822c22638fbeb3a](https://github.com/cockroachdb/cockroach/commits/3a28e789f6739906f0ebe333c822c22638fbeb3a):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-68-g3a28e789f6
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3529552&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3529552&tab=artifacts#/hibernate) on release-21.1 @ [8fdaa9aeb9ee80b467ab6dc883a1659f12132694](https://github.com/cockroachdb/cockroach/commits/8fdaa9aeb9ee80b467ab6dc883a1659f12132694):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-70-g8fdaa9aeb9
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3533726&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3533726&tab=artifacts#/hibernate) on release-21.1 @ [134306f075c20de755a785ada3db896efe0b1d73](https://github.com/cockroachdb/cockroach/commits/134306f075c20de755a785ada3db896efe0b1d73):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/hibernate/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,hibernate.go:169,hibernate.go:182,test_runner.go:733: 
		Tests run on Cockroach v21.1.9-72-g134306f075
		Tests run against hibernate 5.4.30
		412 Total Tests Run
		411 tests passed
		1 test failed
		67 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		18 tests expected failed but not run
		---
		--- FAIL: org.hibernate.userguide.pc.WhereTest.testLifecycle - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blocklist (hibernateBlockList21_1) is available in the artifacts' hibernate log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh hibernate
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/hibernate_blocklist.go
+++ b/pkg/cmd/roachtest/tests/hibernate_blocklist.go
@@ -219,4 +219,6 @@ var hibernateIgnoreList22_1 = hibernateIgnoreList21_2
 
 var hibernateIgnoreList21_2 = hibernateIgnoreList21_1
 
-var hibernateIgnoreList21_1 = blocklist{}
+var hibernateIgnoreList21_1 = blocklist{
+	"org.hibernate.userguide.pc.WhereTest.testLifecycle": "unknown",
+}

